modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/filters/components/Configuration.tsx

```diff
@@ -106,6 +106,7 @@ function Configuration({
 
 	const openSelectFieldModal = () => {
 		openModal({
+			className: 'modal-height-full',
```

Comment:
Nice! Normal `lg` modal had many issues with overflow, did you check if both horizontal and vertical overflows behave as expected?

Reply:
I was concerned about those issues with `lg` but everything seems to be ok (smoke tests + playwright)

![image](https://github.com/user-attachments/assets/76fb7fc2-d4a0-4801-957f-20aa6e35b072)

![image](https://github.com/user-attachments/assets/aa046c73-e98c-4418-837c-3299dc8b6098)